.travis.yml

```diff
@@ -222,10 +222,11 @@ install:
             echo CONFIG_RTE_TOOLCHAIN_${DPDKCC^^}=y >> .config
             popd
             make build O=${TARGET} EXTRA_CFLAGS="-fPIC $DPDK_CFLAGS" CROSS="$DPDK_CROSS" CC="$CC" HOSTCC=gcc -j $(nproc)
+            make install O=${TARGET} DESTDIR=${TARGET}
```

Comment:
If I understand correctly, if we discover that DPDK has been installed, we use it, but if not we download and build a private copy of it. Why would ODP try to install DPDK? Shouldn't that be up to the sysadmin?  The user building ODP may not have the required privileges to install anything. Or are we just talking about a "private installation" here?

Reply:
OK, that makes sense. Thanks.